Sources/MapboxNavigation/Route.swift

```diff
@@ -44,7 +44,6 @@ extension Route {
     }
     
     func congestionFeatures(legIndex: Int? = nil,
-                            isAlternativeRoute: Bool = false,
```

Comment:
Can you make sure that that `isAlternativeRoute` is really not needed? I guess it was used to filter out alternative route lines in `Navigation SDK 1.x`.

Reply:
Yes, Because now the `isAlternativeRoute` is replaced by `isMainRoute`. Now in the source generation `func shape(forCasingOf)` and `func shape(for:)` is using the `isMainRoute` to generate different feature collection. The layer generation function also use the `isMainRoute` to change the color and opacity. So the `isAlternativeRoute` could be removed from the feature properties.